src/app/iteration/iteration-modal/iteration-modal.component.scss

```diff
@@ -33,3 +33,18 @@
 .space-error {
   visibility: visible;
 }
+
+//Toast notification style
+.iteration-toast-notification {
+  position: fixed;
+  top: em(36);
+  right: em(50);
+  z-index: 10;
+  min-width: em(330);
+}
+.toast-iteration-name {
+  max-width: em(80);
+  display: inline-block;
+  vertical-align: top;
```

Comment:
why do we need vertical-align. It doesnot make any difference when we removed it